draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -585,6 +585,8 @@ The server SHOULD delete idle DTLS sessions after an "idle timeout".[^idle-timeo
 
 [^idle-timeout-conf]: RFC 7360 adds a paragraph about that the idle timeout should not be exposed to the admin as configurable parameter and references a mechanism to determine this value from the application-layer watchdog, but I didn't find the specification anywhere.
 
+This session "idle timeout" SHOULD be exposed to the administrator as a configurable setting. It SHOULD NOT be set to less than 60 seconds and SHOULD NOT be set to more than 600 seconds (10 minutes).
```

Comment:
Should be noted that for non-stream setup UDP sockets, Linux's Netfilter uses a timeout of 30 seconds, so might be worth lowering the bottom end to match?